Dockerfile

```diff
@@ -1,2 +1,23 @@
-FROM node:0.10-onbuild
-EXPOSE 3000
\ No newline at end of file
+FROM node:0.10
+
+RUN mkdir -p /opt/wftda/bouttime
+WORKDIR /opt/wftda/bouttime
+
+ARG NODE_ENV
+ENV NODE_ENV $NODE_ENV
+
+RUN echo '{ "allow_root": true }' > /root/.bowerrc
+ADD package.json .
```

Comment:
I've not used shrinkwrap, so I missed it. I take it if we copy that into the working directory it gets picked up automatically?